homeassistant/helpers/entity.py

```diff
@@ -120,6 +120,10 @@ def assumed_state(self):
         """Return True if unable to access real state of entity."""
         return False
 
+    def finalize_setup(self):
```

Comment:
Why is this method needed? Why can't the WeMo device register when the class is initialized?